lib/Paymill/Services/ResponseHandler.php

```diff
@@ -396,25 +479,24 @@ public function convertErrorToModel($response, $resourceName = null)
     }
 
     /**
-     * Validates the data responsed by the API
+     * Validates the data responded by the API
+     * Just checks the header status is successful.
      *
-     * Only Refund, Transaction and Preauthorization return an response_code
      * @param array $response
-     * @return boolean
+     *
+     * @return boolean True if valid
      */
     public function validateResponse($response)
     {
         $returnValue = false;
-        if ($response['header']['status'] == 200) {
```

Comment:
It is. It won't break existing implementations. It removed warnings if index isn't set and extends the range. Maybe I overlooked something, but what makes it not backwards compatible?